src/gui/torrentcontentmodelfolder.cpp

```diff
@@ -86,9 +86,10 @@ TorrentContentModelItem *TorrentContentModelFolder::child(int row) const
 
 TorrentContentModelFolder *TorrentContentModelFolder::childFolderWithName(const QString &name) const
 {
-    for (TorrentContentModelItem *child : asConst(m_childItems))
+    for (TorrentContentModelItem *child : asConst(m_childItems)) {
         if ((child->itemType() == FolderType) && (child->name() == name))
             return static_cast<TorrentContentModelFolder *>(child);
+    }
     return nullptr;
```

Comment:
@Chocobo1:  It does not look like you converted this one. 

Was that intentional?